src/background/storage.js

```diff
@@ -35,33 +36,30 @@ class Storage {
     try {
       let storagePersistNeeded = false;
       this.local = await browser.storage.local.get();
+      this.preferences = await browser.storage.sync.get();
```

Comment:
Personally I like explicit over implicit, so I'd actually prefer `this.sync` here - which would as a side-effect make it possible that we could store something else besides preferences into sync without having do implement a migration-path again.

But since I'm not sure what else could be stored to sync, I'd also be OK with leaving it as it is.